### DISCUSSION TITLE
[VOTE] Release Apollo version 1.9.1

### DISCUSSION THREAD

**Discussion by nobodyiam** [apolloconfig/apollo#3967] [General]
Hi All,

This is a call for vote to release Apollo version 1.9.1

Release notes:

 * https://github.com/ctripcorp/apollo/releases/tag/v1.9.1

Release Candidate:

 * https://github.com/ctripcorp/apollo/releases/download/v1.9.1/apollo-configservice-1.9.1-github.zip
 * https://github.com/ctripcorp/apollo/releases/download/v1.9.1/apollo-adminservice-1.9.1-github.zip
 * https://github.com/ctripcorp/apollo/releases/download/v1.9.1/apollo-portal-1.9.1-github.zip
 * sha1 checksums
   - d6248f187d9e62bbde210061911836bf815d4f8a  apollo-configservice-1.9.1-github.zip
   - 786930152a31ea303f3dc999365a14f5d521dbda  apollo-adminservice-1.9.1-github.zip
   - 585f6159445491c4c8c2539f0b895f8b8fe72fc3  apollo-portal-1.9.1-github.zip

Maven 2 staging repository:

- https://oss.sonatype.org/service/local/repositories/comctrip-1025/content/com/ctrip/framework/apollo/

Release Tag :

 * https://github.com/ctripcorp/apollo/releases/tag/v1.9.1

Release CommitID :

 * https://github.com/ctripcorp/apollo/commit/501e4b95db4f37fba411266ff24e7d7f460649b8

Guide to build the release from source:

 * https://www.apolloconfig.com/#/zh/deployment/distributed-deployment-guide?id=_2212-%e9%80%9a%e8%bf%87%e6%ba%90%e7%a0%81%e6%9e%84%e5%bb%ba

Voting will start now (2021/09/08) and will remain open for at least 72 hours, Request all PMC members and committers to give their vote.
[ ] +1 Release this package.
[ ] +0 No opinion.
[ ] -1 Do not release this package because....

**Comment by Anilople** [comment]
+1

**Comment by klboke** [comment]
+1

**Comment by Accelerator96** [comment]
+1

**Comment by zouyx** [comment]
+1

Checked
[√] All uts passed.
[√] Release Note.
[√] Hash is fine .
[√] Download links are fine.

Zhuohao Li ***@***.***> 于2021年9月8日周三 下午10:14写道：

**Comment by kezhenxu94** [comment]
Hi, the maven staging repo link is not available, if it's addressed I'm +1

- [x] sha sum is correct.
- [x] release tag is correct.
- [x] tag and commit hash match.
- [x] the maven 2 staging repo link is not available

**Reply by nobodyiam** [reply]
I just tested with curl and it looks okay to me...Do we need to post a link to some specific pom or jar here?

> curl https://oss.sonatype.org/service/local/repositories/comctrip-1025/content/com/ctrip/framework/apollo/



```
<content>
  <data>
    <content-item>
      <resourceURI>https://oss.sonatype.org/service/local/repositories/comctrip-1025/content/com/ctrip/framework/apollo/apollo-openapi/</resourceURI>
      <relativePath>/com/ctrip/framework/apollo/apollo-openapi/</relativePath>
      <text>apollo-openapi</text>
      <leaf>false</leaf>
      <lastModified>2021-09-08 12:29:45.0 UTC</lastModified>
      <sizeOnDisk>-1</sizeOnDisk>
    </content-item>
    <content-item>
      <resourceURI>https://oss.sonatype.org/service/local/repositories/comctrip-1025/content/com/ctrip/framework/apollo/apollo-client/</resourceURI>
      <relativePath>/com/ctrip/framework/apollo/apollo-client/</relativePath>
      <text>apollo-client</text>
      <leaf>false</leaf>
      <lastModified>2021-09-08 12:29:41.0 UTC</lastModified>
      <sizeOnDisk>-1</sizeOnDisk>
    </content-item>
    <content-item>
      <resourceURI>https://oss.sonatype.org/service/local/repositories/comctrip-1025/content/com/ctrip/framework/apollo/apollo-mockserver/</resourceURI>
      <relativePath>/com/ctrip/framework/apollo/apollo-mockserver/</relativePath>
      <text>apollo-mockserver</text>
      <leaf>false</leaf>
      <lastModified>2021-09-08 12:29:48.0 UTC</lastModified>
      <sizeOnDisk>-1</sizeOnDisk>
    </content-item>
    <content-item>
      <resourceURI>https://oss.sonatype.org/service/local/repositories/comctrip-1025/content/com/ctrip/framework/apollo/apollo/</resourceURI>
      <relativePath>/com/ctrip/framework/apollo/apollo/</relativePath>
      <text>apollo</text>
      <leaf>false</leaf>
      <lastModified>2021-09-08 12:29:25.0 UTC</lastModified>
      <sizeOnDisk>-1</sizeOnDisk>
    </content-item>
    <content-item>
      <resourceURI>https://oss.sonatype.org/service/local/repositories/comctrip-1025/content/com/ctrip/framework/apollo/apollo-core/</resourceURI>
      <relativePath>/com/ctrip/framework/apollo/apollo-core/</relativePath>
      <text>apollo-core</text>
      <leaf>false</leaf>
      <lastModified>2021-09-08 12:29:32.0 UTC</lastModified>
      <sizeOnDisk>-1</sizeOnDisk>
    </content-item>
  </data>
</content>
```

**Reply by kezhenxu94** [reply]
OK, it looks good to me now

**Comment by nisiyong** [comment]
+1 

I've checked
- [x] shasum are correct for each zip file
- [x] all unit tests are passed for two sources files
- [x] tag and commit hash match

BTW, I wonder how to check the deployed jars from the maven staging repo.

**Reply by nobodyiam** [reply]
After we deploy the jars to maven staging repository, there will be a new repository created, e.g. this time it is `comctrip-1025`, then we could add the repository info in pom.xml. And then maven will load the jars from this repository.
```xml
    <repositories>
        <repository>
            <id>sonatype-nexus-release</id>
            <url>https://oss.sonatype.org/service/local/repositories/comctrip-1025/content</url>
            <releases>
                <enabled>true</enabled>
            </releases>
            <snapshots>
                <enabled>true</enabled>
            </snapshots>
        </repository>
    </repositories>
```

**Reply by nisiyong** [reply]
Thanks, it works! I've checked
- [x] the apollo deployed jars could be fetched from the maven repository

**Comment by nobodyiam** [comment]
The vote to release Apollo version 1.9.1 has passed with 6 +1 and zero -1.

**Reply by nisiyong** [reply]
> The vote to release Apollo version 1.8.2 has passed with 6 +1 and zero -1.

1.8.2? NOT 1.9.1?
